src/wizard/rust/profiling-onboarding/rust/0.alert.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Rust
+doc_link: https://docs.sentry.io/platforms/rust/profiling/
+support_level: alpha
+type: language
+---
+
+<div class='alert warning'>
+Profiling for Rust is currently in Alpha, and there may be some bugs. We recognize the irony. If you have any questions or feedback, join our <a href="https://discord.gg/zrMjKA4Vnz">discord channel</a>.
```

Comment:
Is there some inside joke about irony here that I don’t know yet? :-D

Reply:
This is a "template" warning we're using across all our SDKs that are currently in alpha.

The irony lies in the fact that we're using a profiler to catch performance issue and/or bugs, but the profiler itself being in alpha could have bugs/perf. issues. I find it pretty ironic indeed.